source/includes/_tasks.md

```diff
@@ -75,9 +75,9 @@ print(response.text)
 > Sample request:
 
 ```shell
-curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" -H "X-Api-Key: YOUR API KEY HERE" -d '{
+curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -H "X-Api-Key: YOUR API KEY HERE" -d '{
     "sort_by_field": "task_created_at",
-    "per_page": 200,
+    "per_page": 100,
```

Comment:
The limit was updated here https://github.com/apolloio/leadgenie/commit/166b60d7#diff-1fb6666b50c65a299528e94c806542fc138ac9dea1e95862b3ca9105d31ce34dR342

Using 200 results in an error in the response.